configs/components/openssl.rb

```diff
@@ -54,6 +66,23 @@
     ldflags = ''
     cflags = "$${CFLAGS} -static-libgcc"
     pkg.environment "CC" => "/opt/pl-build-tools/bin/gcc"
+  elsif platform.is_windows?
+    if platform.architecture == "x64"
+      target = "mingw64"
+      arch = "64"
+    else
+      target = "mingw"
+      arch = "32"
+    end
```

Comment:
More consistency really - functionally doesn't make a difference. You definitely don't need a ternary on line 116+ as it would make it unreadable.